selftests/functional/test_interrupt.py

```diff
@@ -104,7 +104,10 @@ def wait_until_no_badtest():
                         if old_psutil:
                             cmdline_list = psutil.Process(p).cmdline
                         else:
-                            cmdline_list = psutil.Process(p).cmdline()
+                            try:
+                                cmdline_list = psutil.Process(p).cmdline()
+                            except psutil.AccessDenied:
+                                continue
```

Comment:
So `cmdline_list` has to be initialized to not explode the line 108/111 below. Or use the same approach as in new line 168.